confmap/provider/s3provider/provider.go

```diff
@@ -31,10 +32,8 @@ import (
 )
 
 const (
-	schemeName   = "s3"
-	stop         = "."
-	colonSlashes = "://"
-	s3URITail    = ".amazonaws.com/"
+	schemeName         = "s3"
+	expectedURIPattern = "s3://(.*)\\.s3\\.(.*).amazonaws\\.com/(.*)"
```

Comment:
Again, doesn't make too much sense to define a global const that is only used once inside a specific function

Reply:
Thanks this comment, fixed~ :)